config/checkstyle/checkstyle.xml

```diff
@@ -177,8 +177,8 @@
       <property name="arrayInitIndent" value="4"/>
     </module>
     <module name="AbbreviationAsWordInName">
-      <property name="ignoreFinal" value="false"/>
-      <property name="allowedAbbreviationLength" value="1"/>
+      <property name="ignoreFinal" value="true"/>
+      <property name="allowedAbbreviationLength" value="3"/>
```

Comment:
this can be reverted now

Reply:
http://checkstyle.sourceforge.net/config_naming.html#AbbreviationAsWordInName